src/QuizData.ExcelReporter/ExcelReporter.cs

```diff
@@ -0,0 +1,241 @@
+﻿using System.IO;
+using System.Linq;
+using OfficeOpenXml;
+using QuizData.Analyser.Models;
+using System.Collections.Generic;
+using OfficeOpenXml.Drawing.Chart;
+
+namespace QuizData.ExcelReport
+{
+    public class ExcelReporter
```

Comment:
Needs a test coverage. md5 checksum advised.

Reply:
Completed in [Added one simple unit test for ExcelReporter](https://github.com/vkamiansky/analysis-practice-jul-2019/commit/c9bb20593e1bc9192a8276e26a4d5fee43d368a4)